ecommerce/views.py

```diff
@@ -8,6 +8,23 @@ class Home(View):
     def get(self,response):
         return render(response,'index.html')
 
+
+class Products(View):
+    def get(self,response):
+        return render(response, 'products.html')
+
+class SingleProduct(View):
+    def get(self,response)
```

Comment:
you should add double dots at the end of get method

Reply:
Solved!